OpenDreamClient/Input/ContextMenu/VerbMenuPopup.xaml.cs

```diff
@@ -23,13 +23,7 @@ internal sealed partial class VerbMenuPopup : Popup {
 
         _verbSystem = verbSystem;
         _target = target;
-
-        if (entityMetaData != null && !string.IsNullOrEmpty(entityMetaData.EntityDescription)) {
-            DescLabel.Margin = new Thickness(4, 0, 4, 0);
-            DescLabel.Text = entityMetaData.EntityDescription;
-        } else {
-            Desc.Visible = false;
-        }
+        Desc.Visible = false;
```

Comment:
The Desc control should be removed from `VerbMenuPopup.xaml` rather than just hidden.